Doc/library/uuid.rst

```diff
@@ -19,8 +19,30 @@ If all you want is a unique ID, you should probably call :func:`uuid1` or
 a UUID containing the computer's network address.  :func:`uuid4` creates a
 random UUID.
 
+Depending on support from the underlying platform, :func:`uuid1` may or may
+not return a "safe" UUID.  A safe UUID is one which is generated using
+synchronization methods that ensure no two processes can obtain the same
+UUID.  All instances of :class:`UUID` have an :attr:`is_safe` attribute
+which relays any information about the UUID's safety, using this enumeration:
 
-.. class:: UUID(hex=None, bytes=None, bytes_le=None, fields=None, int=None, version=None)
+.. class:: SafeUUID
+
+.. attribute:: SafeUUID.safe
+
+   The UUID was generated by the platform in a multiprocessing-safe way.
+
+.. attribute:: SafeUUID.unsafe
+
+   The UUID was not generated in a multiprocessing-safe way.
+
+.. attribute:: SafeUUID.unknown
+
+   The platform does not provide information on whether the UUID was generated
+   safely or not.
+
+.. versionadded:: 3.7
+
+.. class:: UUID(hex=None, bytes=None, bytes_le=None, fields=None, int=None, version=None, *, is_safe=SafeUUID.unknown)
```

Comment:
Currently, the body of `UUID()` documentation briefly describes all its arguments so I think we can just copy is_safe documentation in `Lib/uuid.py`